update-imported-docs/update-imported-docs.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3
+##
+# This script was tested with Python 3.7.4, Go 1.12, and PyYAML 5.1.2
+# installed in a virtual environment.
+#
+# This script updates the generated reference documentation.
+# See https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/
+# for further details.
+#
+# This script checks to make sure Go and PyYAML have been installed.
+# The reference docs are generated by a Go command so the Go binary must be
+# in your PATH.
+#
+# A temp "work_dir" is created and is the path where repos will be cloned.
+# The work_dir is printed out so you can remove it
+# when you no longer need the contents.
+# This work_dir will temporarily become the GOPATH.
+#
+# To execute the script from the website/update-imported-docs directory:
+# ./update-imported-docs.py <config_file>
+# Config files:
+#     reference.yml  use this to update the reference docs
+#     release.yml    use this to auto-generate/import release notes
+##
+
+import argparse
+import glob
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+
+error_msgs = []
+
+reqs = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze'])
```

Comment:
Hey @aimeeu. I am testing the execution of this script on my local linux (Ubuntu) machine.
I did not set up a virtual env and forgot that I do not have pip installed (not default on Ubuntu).
It may be easier just to document the requirements and not programmatically check. The program will quickly fail when the first call is made and a message could explain what needs to be installed?

Reply:
@kbhawkey - thanks for testing and your feedback! I admit to "cheating" with my virtual environment - I installed Anaconda last year when I did an AI nanodegree. So I used `conda` to install the packages I needed, not pip.  I'll look into NOT programmatically checking. In my past life, we just listed prereqs in the comments, and if the user didn't read the comments....

Reply:
I added a check for `pip` to the script and as a requirement in the docs.